.ci/ciao_down_setup.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This setup will be used to run the CI tests under ciao-down
+# under SemaphoreCI. This will avoid issues with the old packages
+# that comes in the SempahoreCI environment (Ubuntu 14.04). 
+
+set -e
+
+cidir=$(dirname "$(readlink -f "$0")")
+
+if [ -z "$LOCALCI_REPO_SLUG" ] && [ -z "$SEMAPHORE_REPO_SLUG" ]; then
+	echo "If you are running locally, please set environment variable
+		LOCALCI_REPO_SLUG for the repo you want to test. For example:
+		export LOCALCI_REPO_SLUG=clearcontainers/tests
+		If you want to test an specific PR, please also add env variable:
+		LOCALCI_PR_NUMBER, for example:
+		export LOCALCI_PR_NUMBER=100" && exit 1
+fi
+
+ciao_repo="github.com/01org/ciao"
+
+# This function will clone the repo inside the VM and
+# get the current PR that will be tested.
+function install_repo_inside_vm(){
+	export $(cat "${HOME}/ci_environment")
+	repo_slug="$LOCALCI_REPO_SLUG"
+	pr_number="$LOCALCI_PR_NUMBER"
+	if [ -n "$SEMAPHORE" ]; then
+		repo_slug="$SEMAPHORE_REPO_SLUG"
+		pr_number="$PULL_REQUEST_NUMBER"
+	fi
+	go get "github.com/${repo_slug}"
+	if [ -n "$pr_number" ]; then
+		pushd "${GOPATH}/src/github.com/${repo_slug}"
+		git fetch origin "refs/pull/${pr_number}/head:pull_${pr_number}"
```

Comment:
if "$pr_number" is empty, it will not go inside the `if` in the line 47, then it will test master branch,